db/migrate/20160203145841_add_content_id_to_about_page.rb

```diff
@@ -0,0 +1,16 @@
+require "securerandom"
+
+class AddContentIdToAboutPage < ActiveRecord::Migration
+  def up
+    add_column :about_pages, :content_id, :string
+
+    AboutPage.reset_column_information
+    AboutPage.all.each do |about_page|
+      about_page.update_attribute(:content_id, SecureRandom.uuid)
```

Comment:
There seems to be a mixture of how we're doing these kind of things. Either atomically in the migration to add the column as you've done here, or separately in a data migration. It might make sense for us to establish a pattern in our approach, at some point.